app/views/govuk_component/tasklist.raw.html.erb

```diff
@@ -0,0 +1,84 @@
+<%
+  steps ||= false
+  small ||= false
+  heading_level ||= 2
+  open_section ||= false
+  remember_last_section ||= false
+
+  section_count = 0
+%>
+<% if steps %>
+  <div
+    data-module="tasklist"
+    class="pub-c-tasklist js-hidden <% if not small %>pub-c-tasklist--large<% end %>"
+    <% if remember_last_section %>data-remember<% end %>
+    >
+    <ol>
+      <% steps.each_with_index do |step, index| %>
+        <%
+          step_active = false
+          step.each do |this_step|
```

Comment:
How do you enumerate on a single `step`? I find this naming confusing. Why do we have `steps` -> `step` -> `this_step` ?

Reply:
Also, if this whole loop construct exists to determine `step_active`, then a more idiomatic way to achieve this is something like:

```ruby
step_is_active = step.any? do |this_step|
  this_step[:panel_links].to_a.any? { |s| s[:active] }
end
```

Reply:
I knew there must be a cleaner way of doing this. Thank you!